### PR TITLE
added two new files

### PR DESCRIPTION
**Pull request by cloudexpert8899**
added two new files

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is my test file.
```
Comment (devendravannela) [inline text.txt:1]: This is not a standard code.

File: text2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is my text 2 file.
```
Comment (devendravannela) [inline text2.txt:1]: test

**Review by devendravannela — CHANGES_REQUESTED** [review]

File: text.txt (hunk 1 above)
Comment (cloudexpert8899) [reply text.txt:1]: issue fixed

File: text2.txt (hunk 2 above)
Comment (cloudexpert8899) [reply text2.txt:1]: done!

**Review by devendravannela — APPROVED** [review]

**Pull request merged**